securedrop-source/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "securedrop-source"
 version = "0.1.0"
 authors = ["redshiftzero <jen@freedom.press>"]
 edition = "2018"
```

Comment:
If this is going to require 1.56, I'd suggest switching to edition 2021, which has some minor improvements: https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html